src/app/modules/main/chat_section/chat_content/controller.nim

```diff
@@ -99,6 +99,13 @@ method init*(self: Controller) =
     var args = ContactArgs(e)
     self.delegate.onContactDetailsUpdated(args.contactId)
 
+  self.events.on(SIGNAL_MESSAGE_DELETION) do(e: Args):
```

Comment:
@Khushboo-dev-cpp actually this part is unnecessary as long as we are not able do delete pinned message. You can see [here](https://github.com/status-im/status-desktop/blob/d3feb58efe817d5cb2d54231de7c8807326bb6bf/ui/imports/shared/views/chat/MessageContextMenuView.qml#L317) when we are allowed to delete a message. But anyway this doesn't bother anything and I don't mind to keep it (so you decide if you want to keep it here or not).

Reply:
Okay I didn't know about this condition for deleting message. So once a message is pinned by someone else now way to delete, someone could get into trouble :p hahaha, will leave to as it is just in case UI makes an error in calling.